frontends/election-manager/public/fonts/edu-tas-beginner.css

```diff
@@ -0,0 +1,6 @@
+@font-face {
+  font-family: 'Edu TAS Beginner';
+  font-weight: 400 700;
```

Comment:
The font file I'm using here, from Google Fonts, is a "Variable Font File" so it includes both normal and bold and everything in between. Based on what I have seen online, I believe the `font-weight: 400 700;` syntax sets up the range of font weights for which the font can be used, so if you use `font-weight: bold;` anywhere, it will be bold. I have confirmed that works.